hazelcast/src/test/java/com/hazelcast/spi/OperationSerializationTest.java

```diff
@@ -0,0 +1,170 @@
+package com.hazelcast.spi;
+
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.nio.BufferObjectDataInput;
+import com.hazelcast.nio.BufferObjectDataOutput;
+import com.hazelcast.nio.serialization.SerializationService;
+import com.hazelcast.test.HazelcastSerialClassRunner;
+import com.hazelcast.test.HazelcastTestSupport;
+import com.hazelcast.test.annotation.QuickTest;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import java.io.IOException;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(HazelcastSerialClassRunner.class)
+@Category(QuickTest.class)
+public class OperationSerializationTest extends HazelcastTestSupport {
+
+    private SerializationService serializationService;
+
+    @Before
+    public void setup() {
+        HazelcastInstance hz = createHazelcastInstance();
+        serializationService = getNode(hz).getSerializationService();
```

Comment:
You should not start a node just to be able to access `SerializationService`. 
Use `SerializationService serializationService = new DefaultSerializationServiceBuilder().build();`